website/docs/webdev/md_components/page_header_components/showcase_markdown.md

```diff
@@ -2,4 +2,16 @@ import Iframe from '../iframe_component.mdx';
 
 <Iframe id='components-page-header--default' height="500" > </Iframe>
 
-<Iframe id='components-page-header--default&args=image:false' height="500" > </Iframe>
\ No newline at end of file
+<Iframe id='components-page-header--default&args=image:false' height="500" > </Iframe>
+
+### Inverted hero
+
+Inverted Page Header with background image as Hero.
+
+<Iframe id='components-page-header--inverted-hero' height="500" > </Iframe>
```

Comment:
For testing embedded iframe: https://github.com/eea/volto-eea-design-system/blob/a4675685b77ac87ad8f2065a473d9d99f29f590e/website/docusaurus.config.js#L19